kolibri/plugins/media_player/assets/src/views/MediaPlayerIndex.vue

```diff
@@ -207,6 +215,7 @@
     },
     beforeDestroy() {
       clearInterval(this.updateContentStateInterval);
+      this.updateProgress();
```

Comment:
Flagging this issue: https://github.com/learningequality/kolibri/issues/2355

I am not sure what the appropriate fix is for it, tbh, but imagine it would have something to do with a navigation guard in the learn ContentPage.

It might require public methods on the renderer components to do any cleanup.

Reply:
(this is not a request to fix this, just flagging that this is doubling down on something that may be an issue!)